src/cloud/reference/ece-tools-reference.md

```diff
@@ -44,14 +47,19 @@ Available commands:
   dev:git:update-composer  Updates composer for deployment from git.
  env
   env:config:show          Display encoded cloud configuration environment variables
+ error
+  error:show               Display info about error by error id or info about all errors from the last deployment
  module
   module:refresh           Refresh config to enable newly added modules
+ schema
+  schema:generate          Generate the schema dist file
```

Comment:
'dist' - please do not abbreviate words as it is not clear - is this 'distribution'?